client/landing/stepper/declarative-flow/internals/hooks/use-migration-experiment/index.ts

```diff
@@ -0,0 +1,9 @@
+import { useExperiment } from 'calypso/lib/explat';
+
+export function useMigrationExperiment() {
+	const [ , experimentAssignment ] = useExperiment(
+		'calypso_signup_onboarding_site_migration_flow_202501_v1'
```

Comment:
I think we also need to check the flow name because most of the steps are used on all migration flows